File: Java/Algorithm/Loop.java
class Contador {
    public static void main(String[] args) {
        int a = 1;
        while ( a < 11) {
            int b = 1;
            while (b < 11) System.out.printf("%02d - %02d, ", a, b++);
            System.out.println();
            a++;
        }
    }
}

//https://pt.stackoverflow.com/q/286931/101

File: C/Algorithm/Loop.c
#include <stdio.h>  

int main() {
    int a, b;
    printf("Digite a: ");
    scanf("%d", &a);
    printf("Digite b: ");
    scanf("%d", &b);
    int soma = 0;
    for (int i = a; i <= b; i++) {
        soma += i;
        printf("%d ", soma);
    }
}

//https://pt.stackoverflow.com/q/298444/101
